packages/assistant/src/RPA/Assistant/library.py

```diff
@@ -585,7 +589,6 @@ def add_file_input(
         name: str,
         label: Optional[str] = None,
         source: Optional[str] = None,
-        destination: Optional[str] = None,
```

Comment:
These kinds of changes are usually breaking changes. Still, since this is under active dev and the `destination` param was never meant to work, I guess is not that bad.

Reply:
I think the important part here is that it was not documented to do anything and was not supposed to do anything, so the existence of the parameter was a _bug_ here. Or that's how I see it. I guess one could semver bikeshed a bunch about this but yeah it's "not that bad". But also just bumping the major would not be that bad.